tests/test.c

```diff
@@ -544,6 +544,9 @@ struct Test tests[] = {
 
     TEST_CASE(test_code_all_available_loaded),
     TEST_CASE_EXPECTED(test_code_load_binary, 24),
+#ifndef AVM_NO_JIT
```

Comment:
@pguyot This is my best try at fixing build error with [build-and-test (macos-15-intel, 28, -DAVM_DISABLE_JIT=OFF)](https://github.com/atomvm/AtomVM/actions/runs/18185429333/job/51793575084?pr=1814#logs). Can we do anything better than just skipping the test?

Reply:
This is the bug: https://github.com/atomvm/AtomVM/pull/1814/files#r2399909494

Additionally, maybe JIT is not very tolerant to hot code reloading on macOS (the mmap didn't have the proper flags), but it's not like we support hot code reloading yet.